docs/01-programs/02-candy-machine/10-available-guards/02-allow-list.md

```diff
@@ -104,29 +193,67 @@ So why can’t we just verify the Merkle Proof directly within the mint instruct
 This path of the route instruction accepts the following arguments:
 
 - **Path** = `proof`: Selects the path to execute in the route instruction.
+- **Merkle Root**: The Root of the Merkle Tree representing the allow list.
```

Comment:
Not sure whether we need this or not, at least the program does not need this.

Reply:
We are also missing the optional `minter` signer, which can be used when the payer is different than the minter.